grafanalib/cloudwatch.py

```diff
@@ -0,0 +1,194 @@
+"""Helpers to create Cloudwatch-specific Grafana queries."""
+
+import attr
+from attr.validators import instance_of
+
+
+@attr.s
+class CloudwatchTarget(object):
+    """
+    Generates Cloudwatch target JSON structure.
+
+    Grafana docs on using Cloudwatch:
+    https://grafana.com/docs/features/datasources/cloudwatch/
+    AWS docs on Cloudwatch metrics:
+    https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/aws-services-cloudwatch-metrics.html
+
+    :param alias: legend alias
+    :param dimensions: Cloudwatch dimensions dict
+    :param metricName: Cloudwatch metric nam
+    :param namespace: CLoudwatch namespace
+    :param period:Cloudwatch data period
+    :param region: CLoudwatch region
+    :param refId: target reference id
+    :param statistics: Cloudwatch mathematic statistic
+    :param checkParams: If false, disable cloudwatch checks
+    """
+    alias = attr.ib(default="")
+    dimensions = attr.ib(default={}, validator=instance_of(dict))
+    metricName = attr.ib(default="")
+    namespace = attr.ib(default="")
+    period = attr.ib(default="")
+    region = attr.ib(default="default")
+    refId = attr.ib(default="")
+    statistics = attr.ib(default=["Average"], validator=instance_of(list))
+    checkParams = attr.ib(default=True, validator=instance_of(bool))
+
+    def to_json_data(self):
+        if self.checkParams:
+            self.__checkParameters()
+
+        return {
```

Comment:
`dict.update` returns `None` (https://docs.python.org/3/library/stdtypes.html#dict.update) which means that the target json ends up being `null`.